### PR TITLE
Fix documentation link

### PR DESCRIPTION
**Pull request by pbiology**
This PR corrects a link found in two of the bwa modules meta.yml file.
They did incorrectly point to a samtools website. 

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
- [x] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [x] If necessary, include test data in your PR.
- [x] Remove all TODO statements.
- [x] Emit the `versions.yml` file.
- [x] Follow the naming conventions.
- [x] Follow the parameters requirements.
- [x] Follow the input/output options guidelines.
- [x] Add a resource `label`
- [x] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [x] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [x] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [x] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

**Review by mashehu — APPROVED** [review]

File: modules/nf-core/bwa/index/meta.yml [hunk 1]
```diff
@@ -11,7 +11,7 @@ tools:
         BWA is a software package for mapping DNA sequences against
         a large reference genome, such as the human genome.
       homepage: http://bio-bwa.sourceforge.net/
-      documentation: http://www.htslib.org/doc/samtools.html
+      documentation: http://bio-bwa.sourceforge.net/
```
Comment (mashehu) [inline modules/nf-core/bwa/index/meta.yml:14]:
```suggestion
      documentation: https://bio-bwa.sourceforge.net/bwa.shtml
```

File: modules/nf-core/bwa/mem/meta.yml [hunk 2]
```diff
@@ -14,7 +14,7 @@ tools:
         BWA is a software package for mapping DNA sequences against
         a large reference genome, such as the human genome.
       homepage: http://bio-bwa.sourceforge.net/
-      documentation: http://www.htslib.org/doc/samtools.html
+      documentation: http://bio-bwa.sourceforge.net/
```
Comment (mashehu) [inline modules/nf-core/bwa/mem/meta.yml:17]:
```suggestion
      documentation: https://bio-bwa.sourceforge.net/bwa.shtml
```

**Review by mashehu — APPROVED** [review]
maybe this is more precise

File: modules/nf-core/bwa/index/meta.yml (hunk 1 above)
Comment (pbiology) [reply modules/nf-core/bwa/index/meta.yml:14]: Yeah that's fine. But then I'll update all of the others as well so it's all the same.

**Review by mashehu — APPROVED** [review]

**Comment by pbiology** [comment]
I'm not sure how to proceed with these tests which are failing. Any input @mashehu ?

**Review by mashehu — APPROVED** [review]

**Comment by mashehu** [comment]
the conda ones are probably okay to ignore, the sha sum is often not reproducible with conda. Can you try to update the ones for `sapme`? I know you didn't touch the code itself, but it's always good if we try to fix the tests when they somehow start failing

**Pull request merged**